academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "IBC Token Lifecyle"
+order: 99
+description: Inter-chain token transfers
+tag: fast-track
+---
+
+# Lifecycle of an Inter-chain token transfer
+
+One of topics that tends to confuse newcomers to blockchain technology and crypto assets is exact nature of coins and tokens. The terminology implies the existence of physical or logical objects. Newcomers may wonder where there objects are in physical space. 
+
+In reality, there are only ledgers and they are non-local, which is to say they are everywhere and nowhere in particular. Protocols enforce strict rules about the state of the ledgers - rules such as only spending funds one actually has, strictly controlling or fixing the total supply, and so on. 
+
+The *experience* of receiving and sending updates to these ledgers is so similar to that of working with cashlike instruments, and the intricacies of blockchain mechanics are sufficiently complex that these simplified terms are sufficient for most users. A user-friendly, cognitively light experience is essential to adoption. 
+
+So, while inter-chain transfers of tokens may be described as a "token" crossing a "bridge" to arrive on another chain, deep down we know that what *must* be unfolding is a coordination of updates between two chains. 
+
+## Design challenge
```

Comment:
Here I would rather re-work the content to talk about why cross-chain transfers with IBC are so important in the Cosmos ecosystem.